src/exo/LoopIR_scheduling.py

```diff
@@ -3808,7 +3778,7 @@ def guard_wrapper(body):
         # the forwarded version. However, in all the current callees, the
         # statement would have been just constructed and if you try to forward
         # you get an error.
-        ir, fwd_wrap = ctxt_stmt_c.parent().body()._wrap(guard_wrapper, "body")
+        ir, fwd_wrap = ctxt_stmt_c.as_block()._wrap(guard_wrapper, "body")
```

Comment:
I think this change is from another PR

Reply:
Yeah, when I merge them inorder. This will go away from this PR

Reply:
sg